cmd/sealer/cmd/alpha/mount.go

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/sealerio/sealer/common"
+	imagecommon "github.com/sealerio/sealer/pkg/define/options"
+	"github.com/sealerio/sealer/pkg/imageengine"
+	osi "github.com/sealerio/sealer/utils/os"
+	"github.com/sealerio/sealer/utils/platform"
+	"github.com/spf13/cobra"
+)
+
+var longMountCmdDescription = `mount image file to mount dir(/var/lib/sealer/data/linux_amd64_)`
+
+var exampleForMountCmd = `
+  sealer alpha mount my-image
+  sealer alpha mount e5b4997adba6
+`
+
+func NewMountCmd() *cobra.Command {
+	mountCmd := &cobra.Command{
+		Use:     "mount",
+		Short:   "mount image file",
+		Long:    longMountCmdDescription,
+		Example: exampleForMountCmd,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			defaultPlatform := platform.GetDefaultPlatform()
+			path := defaultPlatform.OS + "_" + defaultPlatform.Architecture + "_" + defaultPlatform.Variant
```

Comment:
Use a common function to gen path.

Reply:
kakazhou is modify the mount dir, if his work done, the path should be different?

Reply:
The specified installation directory is used here and is not loaded back to the mount directory